packages/react/src/ProgressivelyProvider.tsx

```diff
@@ -39,6 +39,9 @@ export const ProgressivelyProvider = ({
     sdk
       .loadFlags({ ctrl })
       .then((res) => {
+        // To remove, this is for demo purpose
+        throw new Error("zaezae");
+
```

Comment:
Since `loadFlags` handle error silently, we'll never go to the catch step, except if we throw inside then.
That's why I added that for demo purpose.